CIME/config.py

```diff
@@ -180,7 +180,7 @@ def __init__(self):
         self._set_attribute(
             "gpus_use_set_device_rank",
             True,
-            desc="If set to `True` and NGPUS_PER_NODE > 0 then `$RUNDIR/set_device_rank.sh` is appended when the MPI run command is generated.",
+            desc="If set to `True` and NGPUS_PER_NODE > 0 then `$RUNDIR/set_device_rank.sh` or `get_local_rank` (a global script on Derecho/Gust) is appended when the MPI run command is generated.",
```

Comment:
Is there a more general way to do this?  One that would apply to any system and generic script name?

Reply:
Maybe we could copy the `get_local_rank` script on Derecho and save it under the `ccs_config_cesm/machines` folder?

Reply:
I was thinking more along the lines of a new xml variable: MPIEXEC_HELPER_SCRIPT=
or something like that.

Reply:
Oh I see. So add `MPIEXEC_HELPER_SCRIPT` in CMEPS and use it in CIME like what we did for `GPU_TYPE`?

Reply:
@jedwards4b I just added a new XML variable named `MPI_GPU_WRAPPER_SCRIPT` in https://github.com/ESMCI/ccs_config_cesm/pull/97 and https://github.com/ESCOMP/CMEPS/pull/363, and update the settings in CIME here as well. I just built a case on Derecho and the `preview_run` indicated that the `get_local_rank` script was attached to the MPI command line as expected. See the output below:
 ```
   MPIRUN (job=case.run):
      mpiexec  --label  --line-buffer  -n 64 get_local_rank /glade/derecho/scratch/sunjian/F2000dev.f19_f19_mg17.derecho.nvhpc.pumas/bld/cesm.exe   >> cesm.log.$LID 2>&1 
```